src/sqlite-xamarin/CMakeLists.txt

```diff
@@ -1,5 +1,10 @@
 cmake_minimum_required(VERSION 3.6.0)
 
+#
+# MUST be included before project()!
+#
+include("../../build-tools/cmake/xa_common.cmake")
```

Comment:
I suspect that this is basically a no-op, as there is no `libsqlite3_xamarin.dylib` in my build tree.  I think this is only built for Android.